Missionframework/functions/ui/fn_overlayUpdateResources.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_components.hpp"
+/*
+    File: fn_overlayUpdateResources.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2020-05-01
+    Last Update: 2020-05-08
+    License: MIT License - http://www.opensource.org/licenses/MIT
+
+    Description:
+        Update resources overlay.
+
+    Parameter(s):
+        _localVariable - TODO Description [DATATYPE, defaults to DEFAULTVALUE]
+
+    Returns:
+        Resources overlay visible [BOOL]
+*/
+
+params [
+    ["_overlay", displayNull, [displayNull]],
+    ["_show", true, [true]],
+    ["_updateValues", true, [true]],
+    ["_resourceArea", "", [""]]
+];
+
+if (isNull _overlay) exitWith {false};
+if (!_show) exitWith {
+    {
+        (_overlay displayCtrl _x) ctrlShow false;
+    } forEach OVERLAY_RSC_IDCS;
+    false
+};
+
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_FOB) ctrlSetText format ["%1", toUpper _resourceArea]; // todo, local fetched from parent scope
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_SUPPLIES) ctrlSetText format ["%1", (floor KP_liberation_supplies)];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_AMMO) ctrlSetText format ["%1", (floor KP_liberation_ammo)];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_FUEL) ctrlSetText format ["%1", (floor KP_liberation_fuel)];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_UNITCAP) ctrlSetText format ["%1/%2", unitcap,([] call KPLIB_fnc_getLocalCap)];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_HELIPAD) ctrlSetText format ["%1/%2", KP_liberation_heli_count, KP_liberation_heli_slots];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_PLANE) ctrlSetText format ["%1/%2", KP_liberation_plane_count, KP_liberation_plane_slots];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_ALERT) ctrlSetText format ["%1%2", round(combat_readiness),"%"];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_CIVREP) ctrlSetText format ["%1%2", KP_liberation_civ_rep,"%"];
+(_overlay displayCtrl IDC_OVERLAY_RSC_LABEL_INTEL) ctrlSetText format ["%1", round(resources_intel)];
```

Comment:
For all these `format ["%1", value]` things, it would be faster I guess to just use `str value`.
For the `format ["%1/%2", value1, value2]` a `[value1, "/", value2] joinString ""` could be also nicer to read and (even if not noticeable, I guess) gives a little bit better performance.